compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/destructuring-mixed-scope-and-local-variables-with-default.expect.md

```diff
@@ -63,67 +63,63 @@ function useFragment(_arg1, _arg2) {
 }
 
 function Component(props) {
-  const $ = _c(9);
-  let t0;
-  if ($[0] === Symbol.for("react.memo_cache_sentinel")) {
-    t0 = graphql`
+  const $ = _c(8);
+  const post = useFragment(
```

Comment:
this fixture has a locally defined `useFragment` so it isn't using the type definition for the useFragment in shared-runtime.